doc/users/interactive.rst

```diff
@@ -146,45 +141,43 @@ Interactive mode controls:
 
 - whether created figures are automatically shown
 - whether changes to artists automatically trigger re-drawing existing figures
-- whether `.pyplot.show` returns immediately or after all of the
-  figures have been closed when given no arguments
-
+- when `.pyplot.show()` returns: immediately, or after all of the figures have been closed when given no arguments
 
-If in interactive mode, then:
+If in interactive mode:
 
-- newly created figures will be shown immediately
-- figures will automatically redraw on change
-- pyplot.show will return immediately by default
+- newly created figures will be displayed immediately
+- figures will automatically redraw when elements are changed
+- `pyplot.show()` displays the figures and immediately returns
 
-If not in interactive mode then:
-
-- newly created figures and changes to figures will
-  not be reflected until explicitly asked to be
-- pyplot.show runs the GUI event loop and does not return until all of
-  the plot windows are closed
+If not in interactive mode:
 
+- newly created figures and changes to figures are not displayed until 
+   * `.pyplot.show()` is called
+   * `.pyplot.pause()` is called
+   * ` fig.canvas.flush_events` is called
```

Comment:
This is not a reference, so should be doubled ticks.

Reply:
Why aren't these failing the build?

Reply:
The answer to my question is that there is a leading space on the third bullet point which means it was neither a literal or a reference.